src/containers/Tenant/Diagnostics/Healthcheck/Healthcheck.scss

```diff
@@ -35,10 +43,16 @@
     }
 
     &__preview-title {
+        display: flex;
+        align-items: center;
+
         font-weight: 600;
     }
 
     &__self-check-status-indicator {
+        display: flex;
+        align-items: center;
```

Comment:
no reason for these elements to be flex, and the alignment should be baseline, not center
i'd rather 
```scss
.healthcheck {
    &__status-wrapper {
        ...
        align-items: baseline;
    }
}
```